examples/dt/neutron/3cont_2comp_3net-ipv4-geneve-composable-vlan_provider_network

```diff
@@ -0,0 +1,72 @@
+# Deployed Topology 3cont_2comp_3net-ipv4-geneve-composable-vlan-provider-network
+
+## General information
+
+| Revision | Change              |  Date      |
+| -------: | :------------------ | :-----:    |
+|     v0.1 | Initial publication | 2024-01-16 |
+|     v0.2 | Refreshed doc       | 2024-02-29 |
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | vm    | 2      |
+| Networker                                  | vm    | 3      |
+| CFME-tester                                | vm    | 1      |
+
+
+## Services, enabled features and configurations
+
+| Service   | configuration                | Lock-in coverage?  |
+| --------- | ---------------------------- | ------------------ |
+| Neutron   | ML2/OVN                      | Must have          |
+| Glance    | default                      | Must have          |
+| Cinder    | default                      | Must have          |
+| Keystone  | default                      | Must have          |
+| TLS       | default                      | Must have          |
+| Compute   | DifferentHostFilter          | Must have          |
+
+
+### Additional information 
+tenant_network = geneve
+public_network = vlan
+This DT has 3 networker nodes
+This DT needs a CFME-tester node to act as a router
+This DT is mapped to unified job - uni-eta
+
+#### Nova
+
+#### Cinder
+
+#### OVN
+tenant_network_mtu = 9000
```

Comment:
Note, all this enlarged mtu related stuff will require some additional settings in other operators. While initially we planned to keep this setting, now I am thinking that for probably for simplicity we can use a default mtu in unified jobs and just decrease external (aka public or provider) network mtu to some lower value (e.g. 1300). This will fulfill our needs for regression testing of path MTU discovery tests (test_mtu.py in d/s tempest_neutron_plugin plugin or it's corresponding code in whitebox-neutron-tempest-plugin). IMO, we can omit testing jumbo MTU since nfv team covers that.
I.e. I mean maybe we should consider switching here to use tenant_network_mtu = 1442, external_network_mtu = 1300 . The path_mtu and global_physnet_mtu in neutron settings will be default and can be omitted in this case. Wdyt?

Reply:
Removed the Jumbo MTU config.